apps/www/_blog/2023-04-27-postgres-pluggable-strorage.mdx

```diff
@@ -0,0 +1,61 @@
+---
+title: 'Postgres pluggable storage'
```

Comment:
When I see the title, I think the blog post will be a highly technical one but this is mostly about history.

How about, `"History of Postgres pluggable storage"`?